sdk/search/azure-search-documents/azure/search/documents/_internal/_search_indexing_buffered_sender.py

```diff
@@ -12,7 +12,6 @@
 from ._utils import is_retryable_status_code
 from ._search_indexing_buffered_sender_base import SearchIndexingBufferedSenderBase
 from ._generated import SearchIndexClient
-from ..indexes import SearchIndexClient as SearchServiceClient
```

Comment:
Should we remove the public .ctor in this type so there's only one way to get a buffered sender?

Reply:
For Python, there is no way to force user not to do in this way. 

We only recommend user get the buffer sender client from search_client.get() and we show them in samples or readme. :)